packages/ui-components/src/components/NatAmountInput.jsx

```diff
@@ -39,13 +39,6 @@ const makeNatAmountInput =
       value === null ? '0' : valueString,
     );
 
-    const preventSubtractChar = e => {
```

Comment:
This wasn't being invoked on keypress, and it's also not sufficient for if someone pastes a string like "-5" instead of typing from scratch